tests/ViewTest.php

```diff
@@ -195,6 +195,10 @@ public function testRenderWithoutFileExtension(): void
         $this->assertSame('Test', $view
             ->withDefaultExtension('txt')
             ->render('file'));
+        $this->assertSame(
+            'Test',
```

Comment:
Use different content in files (change `file_put_contents`). It's ensures that the expected file is used.